### DISCUSSION TITLE
Translation loader don't work with laravel 9

### DISCUSSION THREAD

**Discussion by fadymondy** [spatie/laravel-translation-loader#148] [Bugs]
i can't set the Translation loader on laravel 9 I think there is overriding by the main laravel loader any help?

**Comment by gottaloveit** [comment]
Same with me.  

[Ignition output](https://flareapp.io/share/KPg32eeP#F76)

[log_dump.txt](https://github.com/spatie/laravel-translation-loader/files/9738035/log_dump.txt)

excerpt from `config/app.php`
```
        Illuminate\Auth\Passwords\PasswordResetServiceProvider::class,
        Illuminate\Session\SessionServiceProvider::class,
        //Illuminate\Translation\TranslationServiceProvider::class,
        Spatie\TranslationLoader\TranslationServiceProvider::class,
        Illuminate\Validation\ValidationServiceProvider::class,
        Illuminate\View\ViewServiceProvider::class,
```

Do you think it might be because in Laravel 9 the lang files are not in `resources` but in the root of the app?
But if you look at the query log, it doesn't even make it to the database.

**Reply by gottaloveit** [reply]
I'm sorry, you can disregard.  My issue was because I had left the files in the `lang/en` folder but nothing in them, like so:

lang/en/roles.php
```
<?php
return [];
```

so it must have tried to load that.  removing the file everything works.
